### ISSUE TITLE
version 3.4.1 is not installable via pip

### ISSUE THREAD

**Issue by sigsergv** [micheles/decorator#3]
~~~~~
% pip install -v decorator==3.4.1 
Collecting decorator==3.4.1
  Getting page https://pypi.python.org/simple/decorator/
  Starting new HTTPS connection (1): pypi.python.org
  "GET /simple/decorator/ HTTP/1.1" 200 488
  URLs to search for versions for decorator==3.4.1:
  * https://pypi.python.org/simple/decorator/
  Getting page https://pypi.python.org/simple/decorator/
  Analyzing links from page https://pypi.python.org/simple/decorator/
    Found link https://pypi.python.org/packages/source/d/decorator/decorator-3.3.1.tar.gz#md5=a8fc62acd705f487a71bc406e19e0cc6 (from https://pypi.python.org/simple/decorator/), version: 3.3.1
    Found link https://pypi.python.org/packages/source/d/decorator/decorator-3.3.2.tar.gz#md5=446f5165af67eb0fcd8fd28abd259e86 (from https://pypi.python.org/simple/decorator/), version: 3.3.2
    Found link https://pypi.python.org/packages/source/d/decorator/decorator-3.3.3.tar.gz#md5=f5a0227cb1c34a0e7d5b7f9cd2ae3135 (from https://pypi.python.org/simple/decorator/), version: 3.3.3
    Found link https://pypi.python.org/packages/source/d/decorator/decorator-3.4.0.tar.gz#md5=1e8756f719d746e2fc0dd28b41251356 (from https://pypi.python.org/simple/decorator/), version: 3.4.0
    Skipping link http://code.google.com/p/micheles/source/browse/#hg%2Fdecorator (from https://pypi.python.org/simple/decorator/); not a file
    Skipping link http://micheles.googlecode.com/hg/decorator/documentation.html (from https://pypi.python.org/simple/decorator/); unknown archive format: .html
    Skipping link http://micheles.googlecode.com/hg/decorator/documentation.pdf (from https://pypi.python.org/simple/decorator/); unknown archive format: .pdf
    Skipping link http://micheles.googlecode.com/hg/decorator/documentation3.html (from https://pypi.python.org/simple/decorator/); unknown archive format: .html
    Skipping link http://micheles.googlecode.com/hg/decorator/documentation3.pdf (from https://pypi.python.org/simple/decorator/); unknown archive format: .pdf
    Skipping link http://packages.python.org/distribute/ (from https://pypi.python.org/simple/decorator/); not a file
    Skipping link http://science.webhostinggeeks.com/dekorater-modula (from https://pypi.python.org/simple/decorator/); not a file
  Could not find a version that satisfies the requirement decorator==3.4.1 (from versions: 3.3.1, 3.3.2, 3.3.3, 3.4.0)
  Cleaning up...
  No distributions matching the version for decorator==3.4.1
~~~~~

**Comment by micheles** [comment]
This is an issue of PyPI. It says to me that version 3.4.1 is there but it does not allow to download it. You can download the module from GitHub instead, I have already asked support from the PyPI team.

**Comment by micheles** [comment]
I have solved by uploading a version 3.4.2 which is identical to 3.4.1 except for the release number.
